### PR TITLE
Align next button + cards + inputs + overall

### PR DESCRIPTION
**Pull request by DoroNahari**
*Final colors alignment is in process

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
-    <loading :active.sync="isLoading"
+  <v-app id="app">
+        <loading :active.sync="isLoading"
              :is-full-page="true"
              :height="128"
              :width="128"
              :color="isLoadingColor"
              loader="dots">
     </loading>
+    <div v-if="!prompts.length" class="loading">{{ messages.generators_loading }}</div>
```
Comment (tomer-epstein) [inline frontend/src/App.vue]: delete this div

File: frontend/src/App.vue [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
       :stepName="prompts[promptIndex].name"
       :rpc="rpc"
+      @parentShowConsole="toggleConsole"
     />
-    <v-row class="m-0 p-0">
-      <v-col class="navigaiton-row m-0 p-0" lg="3" sm="auto">
-          <Navigation v-if="prompts.length > 0" :promptIndex="promptIndex" :prompts="prompts" />
+    <v-row class="main-row m-0 p-0">
+      <v-col class="left-col m-0 p-0" lg="3" sm="auto">
+        <Navigation v-if="prompts.length" :promptIndex="promptIndex" :prompts="prompts" />
```
Comment (tomer-epstein) [inline frontend/src/App.vue]: why did you revert to 'v-if="prompts.length"' ?

File: frontend/src/App.vue [hunk 3]
```diff
@@ -31,23 +36,27 @@
             @generatorSelected="onGeneratorSelected"
             @stepvalidated="onStepValidated"
           />
-          <div class="navigation" v-if="prompts.length > 0 && !isDone">
-            <b-button class="btn" :disabled="!stepValidated" @click="next">Next</b-button>
-          </div>
+        </v-col>
+          <v-col v-if="prompts.length > 0 && !isDone" class="bottom-right-col" style="height: 4rem;" offset-lg="9" lg="3">
```
Comment (tomer-epstein) [inline frontend/src/App.vue]: we are not consistent! here it is is -> v-if="prompts.length > 0

File: frontend/src/main.js [hunk 4]
```diff
@@ -7,7 +7,6 @@ import 'bootstrap/dist/css/bootstrap.css';
 import 'bootstrap-vue/dist/bootstrap-vue.css';
```
Comment (tomer-epstein) [inline frontend/src/main.js]: is there a need for bootstrap-vue anymore?

File: frontend/src/App.vue [hunk 5]
```diff
@@ -1,23 +1,28 @@
 <template>
-  <div id="app" class="d-flex flex-column yeoman-ui vld-parent">
```
[the remark this answers is not in the document]
Comment (DoroNahari) [reply frontend/src/App.vue]: Done

File: frontend/src/App.vue (hunk 1 above)
Comment (DoroNahari) [reply frontend/src/App.vue]: Done

File: frontend/src/App.vue (hunk 2 above)
Comment (DoroNahari) [reply frontend/src/App.vue]: Done

File: frontend/src/App.vue (hunk 3 above)
Comment (DoroNahari) [reply frontend/src/App.vue]: Done

File: frontend/src/main.js (hunk 4 above)
Comment (DoroNahari) [reply frontend/src/main.js]: Will be handled after

Comment (DoroNahari) [reply frontend/src/main.js]: The cards are still written in bootstrap-vue

File: frontend/src/components/Done.vue [hunk 6]
```diff
@@ -67,4 +67,17 @@ img.imgDone {
...
+  color: #cccccc !important;
+  border-radius: 0px !important;
+  font-size: 0.8rem !important;
+  padding: 0.2rem 0.6rem !important;
+  height: 2rem !important; 
+}
+.jumbo button {
+  background-color: var(--vscode-button-hoverBackground, #1177bb) !important;
+  border-color: var(--vscode-button-hoverBackground, #1177bb) !important;
+}
```
Comment (tomer-epstein) [inline frontend/src/components/Done.vue]: todo: remove after meging pull -> https://github.com/SAP/yeoman-ui/pull/83

**Pull request merged**